repository/repository.go

```diff
@@ -423,3 +430,70 @@ To find more information on specific steps, please see documentation [here](http
 - [ ] QA: Final validation of above PR and tracked through the linked ticket
 - [ ] PJM: Close the milestone in GitHub.
 `
+
+type Author struct {
+	Name  string `json:"name"`
+	Email string `json:"email"`
+	Date  string `json:"date"`
+}
+
+type Commit struct {
+	Author    Author `json:"author"`
+	Committer Author `json:"committer"`
+	Message   string `json:"message"`
+	URL       string `json:"url"`
+}
+
+type CommitResponse struct {
+	SHA         string `json:"sha"`
+	NodeID      string `json:"node_id"`
+	Commit      Commit `json:"commit"`
+	URL         string `json:"url"`
+	HTMLURL     string `json:"html_url"`
+	CommentsURL string `json:"comments_url"`
+}
+
+func CommitInfo(owner, repo, commitHash string, httpClient *http.Client) (*Commit, error) {
```

Comment:
Would it be possible to use the GitHub library here? We already have some stuff built around it, so it wouldn't be adding an additional dependency.